README.md

```diff
@@ -1 +1,348 @@
-# dxFeed .NET API
+#  ![logo1][]![logo2][] dxFeed .NET API 
+
+
+[logo1]: https://github.com/Strotsev/dxfeed-net-api/blob/master/feed_logo.png
```

Comment:
Use the `docs` folder (you could create an `assets` or `images` subdirectory) and relative paths